### ISSUE TITLE
Errors IFD-0017 and IFD-0018 raised incorrectly?

### ISSUE THREAD

**Issue by marhop** [EasyinnovaSL/DPFManager#506]
Hi,

Validating the [attached file](https://github.com/EasyinnovaSL/DPFManager/files/4631248/example.zip) ([source](https://github.com/marhop/literate-binary/blob/master/examples/tiff/bilevel-tiff.md)) with DPFManager 3.5.1 using the default (Baseline TIFF 6.0) profile raises errors IFD-0017 "Incorrect Extra Samples Count" and IFD-0018 "Unnecessary Extra Samples" in IFD1.

However, this file has no ExtraSamples field at all. Which is OK because it has no extra samples. It is a very simple bilevel (black/white) TIFF file with PhotometricInterpretation = 0 and no SamplesPerPixel field because the default value (1) given in the [TIFF 6.0 specification](https://www.adobe.io/content/dam/udp/en/open/standards/tiff/TIFF6.pdf) is appropriate.

To summarize: one sample per pixel (default), one bit per sample (default), no extra samples.

What do you think? Am I missing an error in the file or is this a bug in DPFManager?

Thanks,
Martin
